apidoc/Titanium/Media/VideoPlayer.yml

```diff
@@ -679,7 +625,7 @@ properties:
     summary: Whether or not the receiver allows Picture in Picture playback.
     platforms: [iphone, ipad, macos]
     type: Boolean
-    default: true
+    default: false
```

Comment:
By default it is ‘true’.  If this property is not a valid boolean it will set false otherwise it is true. So need to change a bit change in code [here](https://github.com/appcelerator/titanium_mobile/blob/cccfb261292444e1fbda02d7ad00ba0703e6b711/iphone/Classes/TiMediaVideoPlayerProxy.m#L454)